pkgs/development/python-modules/cwl-utils/default.nix

```
@@ -16,7 +16,7 @@
 
 buildPythonPackage rec {
   pname = "cwl-utils";
-  version = "0.28";
+  version = "0.29";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/common-workflow-language/cwl-utils/blob/c4d4b2f9ad1afbe1d9503375f66f59e835da97fb/setup.py#L26

---

Thanks, updated.